Plugins/Public/balancemagic/Main.cpp

```diff
@@ -81,10 +90,11 @@ sockaddr_in sockaddrFLHook;
 sockaddr_in sockaddrKillposter;
 SOCKET sKillposterSocket = NULL;
 
-map<uint, CLIENT_DATA> mapKillmailData;
+CLIENT_DATA aClientData[250];
```

Comment:
isn't there a constant for this?

Reply:
Every plugin handles client data differently.